app/controllers/timecards_controller.rb

```diff
@@ -40,6 +40,28 @@ def update
     end
   end
 
+  def verify
+    authorize! :edit, Timecard
+    @timecard = Timecard.find(params[:timecard_id])
+    if @timecard.status == 'Verified'
```

Comment:
I think we should turn this conditional around, and test that a timecard has a status of "Unverified". A timecard in an error state is not eligible to be verified. In fact, only an `Unverified` timecard can be verified. All others need to be hand edited.
The error message could be something like `Timecard not unverified`